entrypoint.sh

```diff
@@ -61,7 +61,7 @@ add_mask "${USER_ACCESS_TOKEN}"
   cd $TEMP_CLONE_WIKI_FOLDER
   git add .
   git commit -m "$COMMIT_MSG"
-  git push -f --set-upstream https://$USER_ACCESS_TOKEN@github.com/$REPOSITORY.wiki.git master
+  git push -f --set-upstream https://$USER_ACCESS_TOKEN@github.com/$REPOSITORY.wiki.git main
```

Comment:
> When using this workflow in the repository the default branch is main, it does not publish the document to Github wiki page successfully. `(But it returns success result)`

my concern is whether we should update to throw the error in case of failure first with the wrong `master` branch.

Reply:
@luongvo Let bring it to another issue IMO. We might need another discussion for that issue.

Reply:
Can we use `HEAD` to push the current branch? https://stackoverflow.com/a/23241152
```suggestion
  git push -f --set-upstream https://$USER_ACCESS_TOKEN@github.com/$REPOSITORY.wiki.git HEAD
```

Reply:
This is a nice suggestion, another alternative is maybe defining the branch at the `git init` step with the branch name so we can guarantee the branch name
https://git-scm.com/docs/git-init#Documentation/git-init.txt---initial-branchltbranch-namegt

```sh
git init --initial-branch=main
```

Reply:
This is a nice suggestion, we can try the HEAD and see if it works 👀